src/precompile_includer.jl

```diff
@@ -1,44 +1,3 @@
-should_precompile = true
+include("../deps/SnoopCompile/precompile/linux/1.5/precompile_Plots.jl")
 
-
-# Don't edit the following! Instead change the script for `snoop_bot`.
```

Comment:
I see this but it seems this file is edited before anyway. I don't really see the point of it either.

Reply:
The point was to load different precompile statements for different OSs and julia versions. However, we got spammed with so many automatic snoopcompilebot PRs that we decided to only us julia stable on linux for generating precompile statements. So now it's pointless.

Reply:
@aminya will this change be overwritten again by CompileBot?

Reply:
There have been other manual changes to this file and it didn't get changed, which is why I assumed it was fine.

Reply:
OK, great.